ethereum/eth_accounts.yaml

```diff
@@ -3,34 +3,31 @@ info:
   title: Account Information
   version: '1.0'
 servers:
-  - url: 'https://eth-mainnet.alchemyapi.io/v2'
+  - url: 'https://eth-mainnet.alchemyapi.io/v2/{apiKey}'
```

Comment:
why did we decide to add this in here?

Reply:
I remeber you asked me to add apiKey as server variable removing it from path, just that's why...